Cargo.toml

```diff
@@ -15,11 +15,12 @@ edition = "2018"
 maintenance = { status = "actively-developed" }
 
 [features]
-default = ["toml", "json", "yaml", "ini", "ron", "json5", "convert-case", "async"]
+default = ["toml", "json", "yaml", "ini", "ron", "json5", "jsonc", "convert-case", "async"]
```

Comment:
Note about adding a new format: traditionally, we've enabled all of them by default but #621 would have us re-evaluate that assumption.